provider/core/src/data_provider.rs

```diff
@@ -14,7 +14,19 @@ use crate::resource::ResourceOptions;
 use crate::yoke::trait_hack::YokeTraitHack;
 use crate::yoke::*;
 
-use alloc::rc::Rc;
+#[cfg(not(feature = "sync"))]
+macro_rules! rc_type {
+    () => {
+        alloc::rc::Rc<[u8]>
```

Comment:
Question: Are you able to put `type Foo = Bar` in a `cfg` or do you need to use the macro?